arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1110 @@
+      },
+      "description": "Parameters for creating/updating a saved search"
+    },
+    "CoreSummary": {
+      "properties": {
+        "Status": {
+          "type": "string",
+          "description": "The status of a core summary."
+        },
+        "NumberOfDocuments": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of documents of a core summary."
+        }
+      },
+      "required": [
+        "NumberOfDocuments" 
+      ],
+      "description": "The core summary of a search."
+    },
+    "SearchSort": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the field the search query is sorted on."
+        },
+        "order": {
+          "type": "string",
+          "description": "The sort order of the search."
+        }
+      },
+      "description": "The sort parameters for search."
+    },
+    "SearchMetadataSchema": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name."
+        },
+        "version": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The version."
+        }
+      },
+      "description": "Metadata items for saved search items."
+    },
+    "SearchMetadata": {
+      "properties": {
+        "RequestId": {
+          "x-ms-client-name": "SearchId",
+          "type": "string",
+          "description": "The request id of the search."
+        },
+        "resultType": {
+          "type": "string",
+          "description": "The search result type."
+        },
+        "total": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The total search results."
+        },
+        "top": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of top search results."
+        },
+        "id": {
+          "type": "string",
+          "description": "The id of the search results request."
+        },
+        "CoreSummaries": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CoreSummary"
+          },
+          "description": "The core summaries."
+        },
+        "Status": {
+          "type": "string",
+          "description": "The status of the search results."
+        },
+        "StartTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time for the search."
+        },
+        "LastUpdated": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of last update."
+        },
+        "ETag": {
+          "type": "string",
+          "description": "The ETag of the search results."
+        },
+        "sort": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SearchSort"
+          },
+          "description": "Gets or sets sort."
```

Comment:
Don't use "gets or sets", just describe what the property itself means. Depending on target language, this might not even be generated as a property, so the description would make no sense